### ISSUE TITLE
Unable to Launch The Unspoken

### ISSUE THREAD

**Issue by dud226** [LibreVR/Revive#726]
Hello,
Hopefully I can get some help with this issue. Ever since I updated to the latest revive I have been completely unable to launch the unspoken. I always get a failed to launch shortcut error in steamvr. All the other oculus games I have tried (lone echo, robo revive, mage's tale) work fine. I have so far tried reinstalling the unspoken (twice), revive, and oculus home with no change. I have also tried running steam as an administrator. I haven't been able to inject the game directly because I cannot locate the the actual .exe for the unspoken. If anyone has any suggestions that would be greatly appreciated. Thanks!

**Comment by jaorg1234** [comment]
Oculus games are installed in the C:\Program Files\Oculus\Software\Software (or on whatever drive you've installed the Oculus software to). Have you tried to look there for the game's .exe? There should be a folder saying "insomniac-the-unspoken" or something similar.

**Comment by ricosway2003** [comment]
I am having the same problem, The Unspoken was working just fine then 2 updates ago it stopped. Now as soon as I launch the game the screen goes black a few sec. later I am right back at the home screen. I am not even getting a ERROR code. I did try injecting the exe. but I got the same. Thank you

**Comment by sirkitree** [comment]
Latest alpha worked for me. Try out https://github.com/LibreVR/Revive/releases/tag/1.2.0-beta3

**Comment by Ticklerz** [comment]
This exact same thing is happening to me, my guess is that The Unspoken isn't downloading on our computer and Revive can't find it. I really dont know how to fix this. I'm going to uninstall Oculus App atm i'll tell you if anything happens.

**Comment by Ticklerz** [comment]
Did nothing for me and I really wanted to play this game!

**Comment by Ticklerz** [comment]
Waste of $20

**Comment by Armada651** [comment]
@Ticklerz Did you use the latest beta? https://github.com/LibreVR/Revive/releases/tag/1.2.0-beta3

**Comment by Ticklerz** [comment]
yeep

**Comment by Ticklerz** [comment]
When I look up videos they all have the installation under Oculus Software Software but mine wants to install them under a different folder and I'm never getting the exe

**Comment by Ticklerz** [comment]
And also for revive to work does the HTC vive folder stuff and revive and Oculus have to all be in the same drive?

**Comment by Ticklerz** [comment]
Because the one difference between people in videos and me is they install the game ez pz then find it in their Oculus Software Software (route) and it's and exe but mine is like hell no and instals it into a different folder and then when I go into it it's not even an exe, it doesnt even install the exe into my computer.

**Comment by Ticklerz** [comment]
I have tried not messing with anything, still doesnt work

**Comment by Ticklerz** [comment]
Yeah I just got another game (Google Earth VR) for oculus and it had an exe and worked, so It's only happening with unspoken!

**Comment by Ticklerz** [comment]
I did it, I just restarted my oculus and re installed the unspoken and made sure that the folder the games were sent to was just under the hard drive folder

**Comment by Ticklerz** [comment]
Nvm it went away after a minute still doesnt work

**Comment by Ticklerz** [comment]
For some reason I install the game, the exe stays but then it just disappears
where could it have gone

**Comment by Ticklerz** [comment]
its so stupid why
does this happen only with this game

**Comment by Ticklerz** [comment]
I've generally lost trust with ever trying to play oculus games again because i dont want to waste money

**Comment by Ticklerz** [comment]
I got the exe but it is not injecting it just does nothing and when I launch it does nothing and now WTF

**Comment by Armada651** [comment]
You should not inject, you need to use the Revive dashboard that you can see when you open the Dashboard in VR.

**Comment by Ticklerz** [comment]
I did use the dashboard but it says its launching it, does nothing and everytime I try to launch it again it says failed to launch

**Comment by Ticklerz** [comment]
before the exe went away I copied it and put it in a different drive, it's staying there but I still cant launch it, the exe just doesnt work even if I have it

**Comment by Armada651** [comment]
> before the exe went away I copied it and put it in a different drive

You can't move Oculus games, they won't launch if they're in a different location. You need to install the game through the Oculus Store, then use the Revive Dashboard to start the game.
